config/config_mgmt.py

```diff
@@ -146,7 +147,9 @@ def sysLog(self, logLevel=syslog.LOG_INFO, msg=None, doPrint=False):
         # log debug only if enabled
         if self.DEBUG == False and logLevel == syslog.LOG_DEBUG:
             return
-        if flags.interactive !=0 and doPrint == True:
+        # always print >Info level msg with doPrint flag
```

Comment:
># always print >Info level msg with doPrint flag [](start = 8, length = 48)

Your comment is different from your code

Reply:
Agree, Error etc. level has less corresponding number than info 👍 .